smithy-model/src/main/java/software/amazon/smithy/model/validation/ValidatorFactory.java

```diff
@@ -39,6 +39,15 @@ public interface ValidatorFactory {
      */
     List<Validator> loadBuiltinValidators();
 
+    /**
+     * Returns a list of decorators.
+     *
+     * @return Returns the loaded decorators.
+     */
+    default List<ValidationEventDecorator> loadBuiltinDecorators() {
```

Comment:
I think this should just be `loadDecorators`. The "builtin" nomenclature was used to differentiate between validators that can be conditionally applied and configured vs validators that are always applied.

Reply:
Updated in the latest commit.